backend/xivdpscalc/character/__init__.py

```diff
@@ -43,6 +43,13 @@ def get_gcd(self) -> float:
         """
         return math.floor(0.25 * (1000 - self.stats[Stats.SPEED].get_multiplier())) / 100
 
+    def get_cast_time(self, base_cast_time) -> float:
```

Comment:
Can you provide a type annotation for `base_cast_time`? I assume it's a `float` here.

Reply:
Uh, am I crazy, or are there no annotations here? Maybe we need another type for `CastTime`?

Reply:
you are not crazy, i am forgetful